src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -136,7 +136,7 @@ public class BlockGUI extends ModElementGUI<Block> {
 	private final JCheckBox canRedstoneConnect = L10N.checkbox("elementgui.common.enable");
 
 	private final JComboBox<String> tintType = new JComboBox<>(
-			new String[] { "No tint", "Grass", "Foliage", "Water", "Sky", "Fog", "Water fog" });
+			new String[] { "No tint", "Grass", "Foliage", "Foliage Birch", "Foliage Spruce", "Foliage Biome", "Water", "Sky", "Fog", "Water fog" });

```

Comment:
No need to capitalize each word. Just the first word of each option.

Reply:
I was following the minecraft naming scheme on that one so people saw it as they would expect from the game, so should I change it ?

Reply:
Yeah, but no. We don't follow what Minecraft does all the time.

Reply:
Changes made, thanks for pointing me in the right direction.